src/modules/collections/services/queries.gql.ts

```diff
@@ -7,6 +7,11 @@ export const FIND_COLLECTIONS_BY_USER = `
 			is_default
 			created_at
 			updated_at
+			ies {
+				intellectual_entity {
```

Comment:
Bart DB has changed this to ie instead of intellectual_entity
![image](https://user-images.githubusercontent.com/1710840/157427720-f6ae4283-6075-4cb8-bfe0-e3e2113effbc.png)
![image](https://user-images.githubusercontent.com/1710840/157427725-2fc28b8c-3258-4efc-ae5c-4b80c890ac34.png)


```suggestion
				ie {
```